cmd/runtimetest/main.go

```diff
@@ -642,6 +644,12 @@ func validate(context *cli.Context) error {
 			err := v.test(spec)
 			t.Ok(err == nil, v.description)
 			if err != nil {
+				if logLevel >= logrus.ErrorLevel {
+					// Skip 'SHOULD', 'MAY' ...
```

Comment:
Maybe `compliance-level` would be better than `rfc-level`?  There are many RFCs, and what we're interested in here is how well the tested-thing complies with the specification.